antarest/study/web/raw_studies_blueprint.py

```diff
@@ -175,14 +180,14 @@ def get_study(
         return Response(content=json_response, media_type="application/json")
 
     @bp.get(
-        "/studies/{uuid}/areas/aggregate/{output_id}",
```

Comment:
The plain deletion of those endoints are a breaking change of the API, without changing it version, which is dangerous.

We must make sure with users that it's really intended, and we must mention it in a special section in the changelog.